$MirOS: ports/emulators/qemu/patches/patch-slirp_tcp_subr_c,v 1.2 2005/05/03 19:30:05 tg Exp $
--- slirp/tcp_subr.c.orig	Wed Apr 27 20:51:43 2005
+++ slirp/tcp_subr.c	Tue May  3 18:04:51 2005
@@ -635,7 +635,7 @@ tcp_emu(so, m)
 	struct mbuf *m;
 {
 	u_int n1, n2, n3, n4, n5, n6;
-	char buff[256];
+	char buff[257];
 	u_int32_t laddr;
 	u_int lport;
 	char *bptr;
